### PR TITLE
Fix cluster-capacity functionality as a pod.

### PR DESCRIPTION
**Pull request by aveshagarwal**
Fix https://bugzilla.redhat.com/show_bug.cgi?id=1535940

@wjiangjay  @sjenning

### CODE REVIEW

**Comment by aveshagarwal** [comment]
Flake: https://github.com/openshift/origin/issues/17897

**Comment by aveshagarwal** [comment]
@sjenning can you tag lgtm?

File: cmd/cluster-capacity/go/src/github.com/kubernetes-incubator/cluster-capacity/Dockerfile [hunk 1]
```diff
@@ -14,10 +14,9 @@
 
 FROM golang:latest
 
-MAINTAINER Dominika Hodovska <dhodovsk@redhat.com>
+MAINTAINER Avesh Agarwal <avagarwa@redhat.com>
 
 COPY hypercc /bin/hypercc
 RUN ln -sf /bin/hypercc /bin/cluster-capacity
 RUN ln -sf /bin/hypercc /bin/genpod
-COPY config/default-scheduler.yaml /config/default-scheduler.yaml
```
Comment (sjenning) [inline cmd/cluster-capacity/go/src/github.com/kubernetes-incubator/cluster-capacity/Dockerfile]: Do we not need this?

Comment (aveshagarwal) [reply cmd/cluster-capacity/go/src/github.com/kubernetes-incubator/cluster-capacity/Dockerfile]: no

Comment (aveshagarwal) [reply cmd/cluster-capacity/go/src/github.com/kubernetes-incubator/cluster-capacity/Dockerfile]: it is used when a customized scheduler config is passed, which should be done via configmap.

Comment (aveshagarwal) [reply cmd/cluster-capacity/go/src/github.com/kubernetes-incubator/cluster-capacity/Dockerfile]: No need to hard code.

Comment (sjenning) [reply cmd/cluster-capacity/go/src/github.com/kubernetes-incubator/cluster-capacity/Dockerfile]: Do we document where this configmap should be mounted inside the container?

Comment (aveshagarwal) [reply cmd/cluster-capacity/go/src/github.com/kubernetes-incubator/cluster-capacity/Dockerfile]: In https://docs.openshift.com/container-platform/3.7/admin_guide/cluster_capacity.html, we have documented how to mount pod spec via configmap but not customized scheduler config as mostly one does not need that. That said, the above documented can be updated with description about how to mount scheduler config  as a ConfigMap in a pod.

**Pull request merged**